### ISSUE TITLE
Specify Default Accounts

### ISSUE THREAD

**Issue by wjmelements** [MetaMask/metamask-extension#2637]
I want to be able to delete an account in my den, or at least specify a default account. Sometimes the selected account will revert to account 1 without any action on my part, which is undesirable because dapps no longer recognize me.

**Comment by danfinlay** [comment]
If you could provide steps for reproducing this bug, it could really help. Sounds pretty strange to start.

**Comment by danfinlay** [comment]
Theory for research: Do we persist selected account to storage? If not, doing that could be an easy fix.

**Comment by wjmelements** [comment]
When I have to unlock Metamask, it always defaults to the autogenerated account that I don't use. The accounts I use are all "LOOSE".

**Comment by lazaridiscom** [comment]
> The accounts I use are all "LOOSE".

I assume that "LOOSE" refers to the imported accounts.

> When I have to unlock Metamask, it always defaults to the autogenerated account that I don't use.

I understand that @wjmelements uses imported accounts, thus `Account 1` is of no use.

But, I just verified on a fresh install (firefox) : MM remembers the last displayed account. I created the mandatory `Account 1`, 3 more hd accounts, and imported 2 accounts. Exit firefox, login to MM => always on last account (be it internal one or imported one).

Now, deletion is already possible for imported accounts, but not for the main account.

(one could argue that `Account 1` should not be create at all, if the user decides so)
